### DISCUSSION TITLE
Adding username and password for security purposes.

### DISCUSSION THREAD

**Discussion by Dvalin21** [Stirling-Tools/Stirling-PDF#236] [Q&A] [answered]
I apologize if this has been asked, but is there environment settings that would make it requiree username and password before accessing the web ui of the app?

**Comment by Frooodle** [comment, marked as answer]
https://github.com/Frooodle/Stirling-PDF#q1-can-you-add-authentication-in-stirling-pdf

edit:    This is now supported in latest release! v0.13.0

**Comment by Dvalin21** [comment]
Thanks for that, I completely missed it. Oh and just a heads up, I noticed that in your examples do docker-cli and compose it shows ALLOW_GOOGLE_VISIBILITY="true", but in the customized app section it says this is disabled by default. I wasn't sure if this was by design or not. However I did switch it to "false" and when doing so portainer gave me an error saying this wasn't allowed

**Reply by Frooodle** [reply]
It is false by default when not provided
Setting it to true is just a commented out example...

But it's odd you setting it to false gave a error.. it works fine for me :o

**Comment by Frooodle** [comment]
@Dvalin21    This is now supported in latest release! v0.13.0

**Comment by Dvalin21** [comment]
Thanks for the update!! Most appropriate it
